gh-87595: mmap.size() now returns the size on Unix for anonymous memory (#24781)
python/cpython: Misc/NEWS.d/next/Library/2021-03-07-16-31-36.bpo-43429.Koa0mf.rst

The :meth:`~mmap.mmap.size` method of the :class:`mmap.mmap` class now
returns the size of an anonymous mapping on both Unix and Windows.
Previously, the size would be returned on Windows and an :exc:`OSError`

> would be raised on Unix.
> Raise :exc:`ValueError` instead of :exc:`OSError` with ``trackfd=False``.

Author: ```suggestion
would be raised on Unix. :exc:`ValueError` is now raised instead of
:exc:`OSError` when ``trackfd=False``.
```
